dts/arc/quark_se_c1000_ss_devboard.dts

```diff
@@ -0,0 +1,35 @@
+/dts-v1/;
+
+#define __SIZE_K(x) (x * 1024)
+
+#if defined(CONFIG_SOC_QUARK_SE_C1000_SS)
+#define DT_FLASH_SIZE		__SIZE_K(152)
+#define DT_SRAM_SIZE		__SIZE_K(24)
+#define DT_DCCM_SIZE		__SIZE_K(8)
+#else
+#error "Flash and RAM sizes not defined for this chip"
```

Comment:
why is this here, if you want the #defines, than remove the #if, #else, #error as it doesn't serve any purpose

Reply:
will change it